version-customer.json

```diff
@@ -0,0 +1,4 @@
+{
+    "version": "1.0.0"
```

Comment:
I think we don't need version-customer for this repo, it should be exist only in our customer repo

Reply:
version customer removed and added to git ignore since it still needed when run on local to avoid `delete and add` or miss pushed